src/frontend/src/pages/FinancePage/FinanceComponents/GenerateReceiptsModal.tsx

```diff
@@ -156,6 +157,29 @@ const GenerateReceiptsModal = ({
               )}
             />
           </FormControl>
+          <FormControl fullWidth>
+            <FormLabel>Receipt Type</FormLabel>
+            <Controller
+              name="refundSource"
+              control={control}
+              render={({ field: { onChange, value } }) => (
+                <Select
+                  value={value}
+                  defaultValue={refundSourceOptions[2]}
```

Comment:
you should set the defaultValues using the defaultValues attribute on the useForm, see an example below (in the reimbursement request form):
<img width="542" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/c15ea799-392e-441b-adc8-d703322935f0">